docs/src/main/sphinx/installation/kubernetes.rst

```diff
@@ -0,0 +1,395 @@
+=============================
+Trino on Kubernetes with Helm
+=============================
+
+`Kubernetes <https://kubernetes.io>`_ is a container orchestration platform that
+allows you to deploy Trino and other applications in a repeatable manner across
+different types of infrastructure. This can range from deploying on your laptop
+using tools like `kind <https://kind.sigs.k8s.io>`_ and
+`Helm <https://helm.sh>`_, to running on a managed Kubernetes service on cloud
+services like
+`Amazon Elastic Kubernetes Service <https://aws.amazon.com/eks>`_,
+`Google Kubernetes Engine <https://cloud.google.com/kubernetes-engine>`_,
+`Azure Kubernetes Service <https://azure.microsoft.com/en-us/services/kubernetes-service>`_,
+and others.
+
+The fastest way to run Trino on Kubernetes is to use the
+`Trino Helm chart <https://github.com/trinodb/charts>`_. Helm is a package
+manager for Kubernetes applications that allows for simpler installation and
+versioning by templating Kubernetes configuration files. This allows you to
+prototype on your local or on-premise cluster and use the same deployment
+mechanism to deploy to the cloud to scale up.
+
+Prerequisites
+-------------
+
+* A Kubernetes cluster with a
+  `supported version <https://kubernetes.io/releases/>`_ of Kubernetes.
+
+  * If you don't have a Kubernetes cluster, you can
+    :ref:`run one locally using kind <running-a-local-kubernetes-cluster-with-kind>`.
+
+* The `kubectl <https://kubernetes.io/docs/tasks/tools/#kubectl>`_ binary.
```

Comment:
Check how we say this elsewhere 

basically you need kubectl and helm installed and available on the PATH on the machine you use to manager the deployment on k8s

Having the binary ... doesnt really mean enough .. I have it on a sd card.. so what..

Reply:
Going to base it off this: https://trino.io/docs/current/connector/postgresql.html

Reply:
Renaming to requirements

Reply:
Also following the next comment